<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMExposureClass</class>
 <widget class="QDialog" name="EMExposureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMExposure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="emexposure.qrc">
    <normaloff>:/EMExposure/Resources/EM.png</normaloff>:/EMExposure/Resources/EM.png</iconset>
  </property>
  <widget class="QSlider" name="verticalSlider_EM">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>79</y>
     <width>19</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_Exposure">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>79</y>
     <width>21</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_EM">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>309</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Exposure">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>309</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_EM2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_EM1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>71</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>262</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_EM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EM</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emexposure.qrc"/>
 </resources>
 <connections/>
</ui>
